src/views/events/Single.vue

```diff
@@ -107,19 +107,19 @@
                   <td>{{ event.application_status | capitalize }}</td>
                 </tr>
                 <tr>
-                  <th>Application period starts</th>
+                  <th>Application period starts <tooltip text="Date and time in your timezone"></tooltip></th>
```

Comment:
Do you think it's possible to move it to a component? Then we can reuse it later, as it has the same text over and over.
Also I'd go with "The date and time are in your local timezone"

Reply:
Should I call the component "tooltip-timezone"?

Reply:
Looks nice to me.
For reference, you can check out `TimezoneNotification` component on event editing (in `src/views/events/Edit.vue`), I've done sth like this here

Reply:
I like this "encapsulation" :)